dev/breeze/src/airflow_breeze/utils/projects_google_spreadsheet.py

```diff
@@ -0,0 +1,253 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from enum import Enum
+from pathlib import Path
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from googleapiclient.discovery import Resource
+
+from airflow_breeze.utils.console import get_console
+
+INTERESTING_OPSF_FIELDS = [
+    "Score",
+    "Code-Review",
+    "Maintained",
+    "Dangerous-Workflow",
+    "Security-Policy",
+    "Packaging",
+    "Vulnerabilities",
+]
+
+INTERESTING_OPSF_SCORES = ["OPSF-" + field for field in INTERESTING_OPSF_FIELDS]
+INTERESTING_OPSF_DETAILS = ["OPSF-Details-" + field for field in INTERESTING_OPSF_FIELDS]
+
+
+class MetadataFromSpreadsheet(Enum):
+    KNOWN_REPUTABLE_FOUNDATIONS = 0
```

Comment:
nit: We can use `auto` to not have to hardcode manually the number increment.
```suggestion
    KNOWN_REPUTABLE_FOUNDATIONS = auto()
```

:warning: auto() starts at 1 I believe so there is a small difference.

Reply:
Well maybe not a good idea, because we rely on these specific value to convert them back, so we might want to be sure those do not change if the enum get updated

Reply:
Yeah. Enums are somewhat broken and even the 3.11's StrEnum and IntEnum have not improved too much.

I would very much prefer if we could use:

```
class MetadataFromSpreadsheet(Enum):
    KNOWN_REPUTABLE_FOUNDATIONS,
    KNOWN_STRONG_COMMUNITIES,
    ....
```

But in this case - yes, we can use auto() - even if it will change the numbers, because we use 

```MetadataFromSpreadsheet[metadata_field]``` which uses enum name.